src/integrationTest/java/uk/gov/hmcts/reform/laubackend/idam/bdd/UserDeletionAuditPostSteps.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.laubackend.idam.bdd;
+
+import io.cucumber.java.DataTableType;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.laubackend.idam.dto.DeletionLog;
+import uk.gov.hmcts.reform.laubackend.idam.request.UserDeletionPostRequest;
+import uk.gov.hmcts.reform.laubackend.idam.response.UserDeletionPostResponse;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static uk.gov.hmcts.reform.laubackend.idam.helper.RestHelper.postObject;
+
+public class UserDeletionAuditPostSteps extends AbstractSteps {
+
+    private String userDeletionResponseBody;
+    private HttpStatus responseStatus;
+
+    @DataTableType
+    public DeletionLog deletionLog(Map<String, String> entry) {
```

Comment:
Do we use deletionLog(Map<String, String> entry) anywhere?

Reply:
Yes, not directly, but yes. Cucumber use this to convert data from data table to list of deletion logs.